bidengine/shellscript.go

```diff
@@ -126,42 +127,11 @@ func parseStorage(resource atypes.Volumes) []storageElement {
 	return res
 }
 
-func (ssp shellScriptPricing) CalculatePrice(ctx context.Context, req Request) (sdk.DecCoin, error) {
-	buf := &bytes.Buffer{}
+func (ssp shellScriptPricing) CalculatePrice(ctx context.Context, req Request, res ctypes.Reservation) (sdk.DecCoin, error) {
```

Comment:
there is no need to pass `ctypes.Reservation` as parameter here
instead, include allocated resourceUnits inside Request and use it instead of `for i, group := range req.GSpec.Resources`